packages/shared/lib/core/wallet/actions/mintNft.ts

```diff
@@ -13,24 +18,32 @@ import {
 import { NftActivity } from '../types'
 import { getDefaultTransactionOptions, preprocessOutgoingTransaction } from '../utils'
 import { generateSingleNftActivity } from '../utils/generateActivity/generateSingleNftActivity'
-import { plainToInstance } from 'class-transformer'
+
+// TODO: Modify this temporary interface
```

Comment:
Shouldn't we maybe link an issue here too?

Reply:
https://github.com/iotaledger/firefly/issues/8134
ready sir ✅

Reply:
he means adding the issue inside the code too in the TODO so you can go back to it https://github.com/iotaledger/firefly/issues/8134

Reply:
something like
```suggestion
// TODO: Update this temporary interface when fixed in the SDK, linked issue https://github.com/iotaledger/firefly/issues/8134
```